spec/enumerables_spec.rb

```diff
@@ -0,0 +1,162 @@
+require './enumerable.rb'
```

Comment:
[OPTIONAL] Using `require_relative` for files in your directory instead of `require` is a ruby convention.

"Require should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory."

And there is no need to add the `.rb` extension again.